gio/Gir.toml

```diff
@@ -1122,6 +1121,18 @@ name = "Gio.SocketAddress"
 status = "generate"
 concurrency = "send+sync"
 
+[[object]]
+name = "Gio.SocketAddressEnumerator"
+status = "generate"
+    [[object.function]]
+    name = "next"
```

Comment:
Please add a comment to both to tell why it's needed.

Reply:
Please add a link to the GLib MR here. The comment is optional, but having the MR link here allows us to get rid of this once we imported updated XML files.

Reply:
Sure, here it is:
https://gitlab.gnome.org/GNOME/glib/-/merge_requests/2701

Reply:
I think he meant in the comment? (please correct me if I'm wrong @sdroege)

Reply:
Yes, inside the `Gir.toml`

Reply:
So I can simply grep for things when updating the XML files